feature-libs/product-configurator/rulebased/components/attribute/types/numeric-input-field/configurator-attribute-numeric-input-field.component.service.ts

```diff
@@ -348,7 +349,7 @@ export class ConfiguratorAttributeNumericInputFieldService {
     currentValue?: string
   ): ValidatorFn {
     return (control: AbstractControl): { [key: string]: any } | null => {
-      const input: string = control.value;
+      const input: string = control.value.trim();
```

Comment:
```suggestion
      const input = control.value?.trim();
```
We later check if input is defined (don't know if control.value is defined in all cases)